go.mod

```diff
@@ -1,3 +1,3 @@
-module github.com/kunalkushwaha/ltag
+module github.com/containerd/ltag
```

Comment:
Maybe we should add also a deprecated comment. https://go.dev/ref/mod#go-mod-file-module-deprecation

Reply:
These are the usages https://github.com/search?type=code&q=kunalkushwaha%2Fltag. Since it has been in containerd/ltag for some time, I think we can do the migration.